File: freecodecamp/daily-code-challenge/2026-02-12.py
def largest_difference(skater1, skater2):
    max_diff = float('-inf')
    lap_number = -1

    for i in range(len(skater1)):
        diff = abs(skater1[i] - skater2[i])

        if diff > max_diff:
            max_diff = diff
            lap_number = i + 1  # Lap numbers are 1-based

    return lap_number

File: freecodecamp/2026-02-12.js
function largestDifference(skater1, skater2) {
  let maxDiff = -Infinity;
  let lapNumber = -1;

  for (let i = 0; i < skater1.length; i++) {
    const diff = Math.abs(skater1[i] - skater2[i]);

    if (diff > maxDiff) {
      maxDiff = diff;
      lapNumber = i + 1; // Lap numbers are 1-based
    }
  }

  return lapNumber;
}
